uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/TOILRequest.php

```diff
@@ -143,8 +139,10 @@ private static function validateValidTOILPastDaysRequest($params) {
    *
    * @param CRM_HRLeaveAndAbsences_BAO_LeaveRequest $leaveRequest
    *   The Leave Request created by this TOIL Request
+   * @param float $toil_to_accrue
+   *   The amount of TOIL to be accrued.
    */
-  private function saveBalanceChange(LeaveRequest $leaveRequest) {
+  private function saveBalanceChange(LeaveRequest $leaveRequest, $toil_to_accrue) {
```

Comment:
can you use camelCase here, to keep it consistent with the other methods params?